packages/katello/katello-host-tools/katello-host-tools.spec

```diff
@@ -165,13 +188,31 @@ exit 0
 
 %files
 %defattr(-,root,root,-)
-%config %{_sysconfdir}/yum/pluginconf.d/package_upload.conf
-%config %{_sysconfdir}/yum/pluginconf.d/enabled_repos_upload.conf
-/var/cache/katello-agent/
+%dir /var/cache/katello-agent/
+%config(noreplace) %{plugins_confdir}/package_upload.conf
+%config(noreplace) %{plugins_confdir}/enabled_repos_upload.conf
+%{katello_libdir}/constants.py*
```

Comment:
Would it make sense to just add ```%{katello_libdir}/*.py*```?

Reply:
I don't think so - tracer.py exists in that folder, and that's installed by the tracer subpackage. Also, I think it's better to be more explicit in the files section so nothing can sneak in but maybe that's just me!

Reply:
I guess I'd organize the package a bit more so tracer is in a separate directory but explicit is good.